lib/data/wrapper.js

```diff
@@ -153,11 +153,16 @@ const data = {
 
     // It would be preferable to have an sproxyd batch delete route to
     // replace this
-    batchDelete: (locations, log) => {
+    batchDelete: (locations, log, noDelete) => {
         // TODO: The method of persistence of sproxy delete key will
         // be finalized; refer Issue #312 for the discussion. In the
         // meantime, we at least log the location of the data we are
         // about to delete before attempting its deletion.
+        const skipIfBackend = ['aws-test'];
+        if (noDelete &&
+        skipIfBackend.indexOf(locations[0].dataStoreName) > -1) {
+            return;
```

Comment:
yeah, with our normal functionality, if an object is put with the same key as an existing object, we delete the old object and put the new one. This doesn't work with AWS though, because we end up deleting the object after the second put request is done, so there's ultimately no object on AWS